docs/conf.py

```diff
@@ -24,6 +24,9 @@
 import django
 from django.utils import six
 
+sys.path.insert(0, os.path.join(os.path.abspath(os.path.dirname(__file__)), '..'))
```

Comment:
Basically, in order to have RTD run sphinx-apidoc automatically for Django code, I need to either do this or include an appropriate settings file (which nothing else would use) with the distributed package.  This hack adds the directories under the project root, including the `schema` package, to the python module search path.